added problem 1 solution

aleno4ka1998 opened the pull request:
wow! it works

aleno4ka1998 committed:
added problem 1 solution

thelightningseas commented on .idea/gentle_lerning.iml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Можно создать файл ```.gitignore```
и написать в него, например
```
./idea/*
```
тогда всякие системные файлы твоей кофигурации локальной не будут добавляться в репозиторий (где они не нужны)

aleno4ka1998 committed:
Правка 6 задачи по серёжиным замечаниям

Требуется определить, бьет ли слон, стоящий на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты слона и координаты другой фигуры. Координаты - целые числа в интервале от 1 до 8.

Выходные данные
Требуется вывести слово YES, если слон способен побить фигуру за 1 ход, в противном случае вывести слово NO

aleno4ka1998 committed:
Правка 6 задачи по серёжиным замечаниям

Правка 6 задачи по серёжиным замечаниям

Требуется определить, бьет ли слон, стоящий на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты слона и координаты другой фигуры. Координаты - целые числа в интервале от 1 до 8.

Выходные данные
Требуется вывести слово YES, если слон способен побить фигуру за 1 ход, в противном случае вывести слово NO

aleno4ka1998 committed:
дай мне сделать реквест, машина...

aleno4ka1998 committed:
все не то и все не так...

aleno4ka1998 committed:
правки в 6 задачу

Входные данные
Даны три целых числа, каждое записано в отдельной строке.

Выходные данные
Выведите наибольшее из данных чисел (программа должна вывести ровно одно целое число).

aleno4ka1998 committed:
Побьёт ли слон фигуру?

Требуется определить, бьет ли слон, стоящий на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты слона и координаты другой фигуры. Координаты - целые числа в интервале от 1 до 8.

Выходные данные
Требуется вывести слово YES, если слон способен побить фигуру за 1 ход, в противном случае вывести слово NO

aleno4ka1998 committed:
максимальное число из двух данных

aleno4ka1998 committed:
високосный ли год?

aleno4ka1998 committed:
Ладья

Требуется определить, бьет ли ладья, стоящая на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты ладьи (два числа) и координаты другой фигуры (два числа), каждое число вводится в отдельной строке. Координаты - целые числа в интервале от 1 до 8.

Выходные данные
Требуется вывести слово YES, если ладья сможет побить фигуру за 1 ход и NO - в противном случае.

aleno4ka1998 committed:
Слон

Требуется определить, бьет ли слон, стоящий на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты слона и координаты другой фигуры. Координаты - целые числа в интервале от 1 до 8.

Выходные данные
Требуется вывести слово YES, если слон способен побить фигуру за 1 ход, в противном случае вывести слово NO

aleno4ka1998 committed:
Ферзь

Требуется определить, бьет ли ферзь, стоящий на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты ферзя и координаты другой фигуры. Координаты - целые числа в интервале от 1 до 8.

Выходные данные
Требуется вывести слово YES, если ферзь может побить фигуру за 1 ход, в противном случае вывести слово NO

aleno4ka1998 committed:
Create please.gitignore

thelightningseas commented on please.gitignore:
```diff
@@ -0,0 +1 @@
+./idea/*
```
файл должен называться просто ```.gitignore```
.гитигнор это не расширение, а название файла, а расширения у него нет

thelightningseas replied:
если по-другому назвать, гит не поймет, что ему надо смотреть в этот файл и удалять то что в нем написано

aleno4ka1998 committed:
Король

Поле шахматной доски определяется парой чисел (a, b), каждое от 1 до 8, первое число задает номер столбца, второе – номер строки. Заданы две клетки. Определите, может ли шахматный король попасть с первой клетки на вторую за один ход.

Входные данные
Даны 4 целых числа от 1 до 8 каждое, первые два задают начальную клетку, вторые два задают конечную клетку. Начальная и конечная клетки не совпадают. Числа записаны в отдельных строках.

Выходные данные
Программа должна вывести YES, если из первой клетки ходом короля можно попасть во вторую, или NO в противном случае.

aleno4ka1998 committed:
rename

aleno4ka1998 committed:
Конь

Требуется определить, бьет ли конь, стоящий на клетке с указанными координатами (номер строки и номер столбца), фигуру, стоящую на другой указанной клетке.

Входные данные
Вводятся четыре числа: координаты коня и координаты другой фигуры. Все координаты - целые числа в интервале от 1 до 8.

Выходные данные
Программа должна вывести слово YES, если конь может побить фигуру за 1 ход, в противном случае вывести слово NO.
